armi/reactor/converters/uniformMesh.py

```diff
@@ -792,12 +734,18 @@ class NeutronicsUniformMeshConverter(UniformMeshGeometryConverter):
     back to the source reactor.
     """
 
-    REACTOR_PARAM_MAPPING_CATEGORIES = [parameters.Category.neutronics]
-    BLOCK_PARAM_MAPPING_CATEGORIES = [
-        parameters.Category.detailedAxialExpansion,
-        parameters.Category.multiGroupQuantities,
-        parameters.Category.pinQuantities,
-    ]
+    REACTOR_PARAMS_TO_MAP = {
+        "in": [parameters.Category.neutronics],
+        "out": [parameters.Category.neutronics],
+    }
+    BLOCK_PARAMS_TO_MAP = {
```

Comment:
I am wondering why the name of this dict was changed? To me, the original name `BLOCK_PARAM_MAPPING_CATEGORIES` was more descriptive of what is actually contained in the dict.

The new name `BLOCK_PARAMS_TO_MAP` indicates to me that it contains parameter names, but this is not the case.

Reply:
I never saw this comment somehow.

No good reason for it other than the order in which development happened. I started by trying to create a list of parameter names, not categories, so I changed the attribute name. And then I went back to categories.

Another good question is why is this snake case? I have no idea. I should probably change it to `blockParamMappingCategories`.